src/main/kotlin/no/nav/dagpenger/regel/minsteinntekt/LøsningService.kt

```diff
@@ -0,0 +1,59 @@
+package no.nav.dagpenger.regel.minsteinntekt
+
+import no.nav.dagpenger.regel.minsteinntekt.Minsteinntekt.Companion.AVTJENT_VERNEPLIKT
+import no.nav.dagpenger.regel.minsteinntekt.Minsteinntekt.Companion.BRUKT_INNTEKTSPERIODE
+import no.nav.dagpenger.regel.minsteinntekt.Minsteinntekt.Companion.FANGST_OG_FISK
+import no.nav.dagpenger.regel.minsteinntekt.Minsteinntekt.Companion.INNTEKT
+import no.nav.dagpenger.regel.minsteinntekt.Minsteinntekt.Companion.LÆRLING
+import no.nav.helse.rapids_rivers.JsonMessage
+import no.nav.helse.rapids_rivers.RapidsConnection
+import no.nav.helse.rapids_rivers.River
+import no.nav.nare.core.evaluations.Evaluering
+import no.nav.nare.core.evaluations.Resultat
+
+class LøsningService(
+    rapidsConnection: RapidsConnection
+) : River.PacketListener {
+    init {
+        River(rapidsConnection).apply {
+            validate { it.requireAll("@behov", listOf(MINSTEINNTEKT)) }
+            validate { it.forbid("@løsning") }
+            validate { it.requireKey("@id", INNTEKT, BEREGNINGSDATO_NY_SRKIVEMÅTE) }
+            validate { it.interestedIn(LÆRLING, FANGST_OG_FISK, AVTJENT_VERNEPLIKT, BRUKT_INNTEKTSPERIODE) }
+        }.register(this)
+    }
+
+    companion object {
+        const val MINSTEINNTEKT = "Minsteinntekt"
+        const val BEREGNINGSDATO_NY_SRKIVEMÅTE = "beregningsdato"
+    }
+
+    override fun onPacket(packet: JsonMessage, context: RapidsConnection.MessageContext) {
```

Comment:
Kunne vært greit å legge på onError også 

Se https://nav-it.slack.com/archives/CCP6QNBSN/p1587749759488600?thread_ts=1587743550.481300&cid=CCP6QNBSN